### PR TITLE
Update command-line-tools.rst

### PR DESCRIPTION
**Pull request by wiersgallak**
Docs for https://github.com/mattermost/mattermost-server/pull/12538

### CODE REVIEW

File: source/administration/command-line-tools.rst [hunk 1]
```diff
@@ -2234,7 +2235,29 @@ mattermost webhook modify-outgoing
           --icon [iconURL]              Icon URL 
 	  --url [callbackURL]           Callback URL 
 	  --content-type string         Content type 
+	  
+mattermost webhook move-outgoing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+  Description
+    Move an existing outgoing webhook to another team and channel by by specifying its id. If the webhook to be moved has a channel id, it must have a new one specified when moving to a new team.
```
Comment (justinegeffen) [inline source/administration/command-line-tools.rst]:
```suggestion
    Move an existing outgoing webhook to another team and channel by specifying its ID. If the webhook you're moving already has a channel ID, a new one must be specified when moving it to the new team. 
```

File: source/administration/command-line-tools.rst [hunk 2]
```diff
@@ -2083,6 +2083,7 @@ mattermost webhook
     -  `mattermost webhook list`_ - List all webhooks.
     -  `mattermost webhook modify-incoming`_ - Modify an existing incoming webhook by changing its title, description, channel or icon url.
     -  `mattermost webhook modify-outgoing`_ - Modify an existing outgoing webhook by changing its title, description, channel, icon, url, content-type, and triggers.
+    -  `mattermost webhook move-outgoing`_ - Move an existing outgoing webhook with an id.
```
Comment (justinegeffen) [inline source/administration/command-line-tools.rst]:
```suggestion
    -  `mattermost webhook move-outgoing`_ - Move an existing outgoing webhook with an ID.
```

File: source/administration/command-line-tools.rst [hunk 3]
```diff
@@ -2083,6 +2083,7 @@ mattermost webhook
     -  `mattermost webhook list`_ - List all webhooks.
     -  `mattermost webhook modify-incoming`_ - Modify an existing incoming webhook by changing its title, description, channel or icon url.
     -  `mattermost webhook modify-outgoing`_ - Modify an existing outgoing webhook by changing its title, description, channel, icon, url, content-type, and triggers.
```
Comment (justinegeffen) [inline source/administration/command-line-tools.rst]:
```suggestion
    -  `mattermost webhook modify-outgoing`_ - Modify an existing outgoing webhook by changing its title, description, channel, icon, URL, content-type, and triggers.
```

File: source/administration/command-line-tools.rst [hunk 4]
```diff
@@ -2083,6 +2083,7 @@ mattermost webhook
     -  `mattermost webhook list`_ - List all webhooks.
     -  `mattermost webhook modify-incoming`_ - Modify an existing incoming webhook by changing its title, description, channel or icon url.
```
Comment (justinegeffen) [inline source/administration/command-line-tools.rst]:
```suggestion
    -  `mattermost webhook modify-incoming`_ - Modify an existing incoming webhook by changing its title, description, channel, or icon URL.
```

File: source/administration/command-line-tools.rst [hunk 5]
```diff
@@ -2234,7 +2235,29 @@ mattermost webhook modify-outgoing
           --icon [iconURL]              Icon URL
```
Comment (justinegeffen) [inline source/administration/command-line-tools.rst]:
```suggestion
         --icon [iconURL]              Icon URL 
```

Comment (justinegeffen) [reply source/administration/command-line-tools.rst]: Need to confirm whether this suggestion does bring the commands into alignment.

File: source/administration/command-line-tools.rst (hunk 1 above)
Comment (justinegeffen) [reply source/administration/command-line-tools.rst]: Question: I may not understand the context but the example below doesn't look like it includes a new ID when moving it to a new channel. Also, is it possible for a webhook not to have an ID (i.e., "if it already has an ID") or are we saying that a webhook has an ID but can also have a channel ID?

Comment (wiersgallak) [reply source/administration/command-line-tools.rst]: @justinegeffen I rewrote the description, does that clarify the requirements on the channel ID for you?  Also updated the example to be more explicit on the webhook ID and channel id/name.  Let me know if this is better?

Comment (justinegeffen) [reply source/administration/command-line-tools.rst]: That's awesome, thank you. :)

**Pull request merged**